components/app/interface/settings/client/index.tsx

```diff
@@ -196,6 +197,14 @@ const Config: SettingsConfiguration<{ server: Server }> = {
           {
             id: "logout",
             icon: <BiSolidExit size={20} color={theme!.colours.error} />,
+            href: "/login",
+            onClick: () => {
+              const acli = getController("state");
+              const cli = getController("client").getCurrentClient();
```

Comment:
client is available through context here, add `const client = useClient()` at the top of the function instead of using this line here.